src/components/core/GeoChart.tsx

```diff
@@ -0,0 +1,239 @@
+// (C) 2007-2019 GoodData Corporation
+import * as React from "react";
+import isEqual = require("lodash/isEqual");
+import noop = require("lodash/noop");
+import mapboxgl = require("mapbox-gl");
```

Comment:
Please try `import { Map } from '"mapbox-gl";`

Reply:
SD-780 will handle this